### PR TITLE
k8s: add helm for local deployments

### LINKED ISSUE #68: k8s: add helm chart for services to remove code duplication in deployment / service spec
api-srv.yaml and book-srv.yaml and any other service is identical except for few details let's use helm for templating.

### PR DESCRIPTION
**Pull request by kevinbarbour**
Moves deployment and services to Helm templates and integrate skaffold
with helm.

fixes #68

### CODE REVIEW

File: k8s/chart/.helmignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (mmatczuk) [inline k8s/chart/.helmignore:1]: What is the purpose of this file here?

**Comment by mmatczuk** [comment]
I'd prefer to have:

- release per service
- helm values in scaffold.yaml not separate file
- env vars as means of configuration

Please do not exaggerate with over configuration, the bottom line is that the deployment files are almost identical for different services and we want to avoid the duplication.

I'd say you need name and replicas, everything else should be changed in defaults or kept in the template.

File: k8s/chart/.helmignore (hunk 1 above)
Comment (kevinbarbour) [reply k8s/chart/.helmignore:1]: Generated by helm when you create a chart, but we don't use the package feature so it's no good to us. Will delete.

File: skaffold.yaml [hunk 2]
```diff
@@ -9,7 +9,21 @@ build:
...
+        containerPort: 5080
+    - name: api
+      chartPath: ./k8s/chart
+      artifactOverrides:
+        image: localhost/sc-poc
+      overrides:
+        replicas: 1
+        env:
+          - name: SCYLLA_CLOUD_BOOK_ADDR
+            value: http://book-svc:5080
```
Comment (mmatczuk) [inline skaffold.yaml:29]: Change default in the command.

**Pull request merged**